code/modules/antagonists/blob/blob_abilities.dm

```diff
@@ -1119,6 +1122,13 @@
 			repeatable--
 		if (repeatable == 0)
 			owner.available_upgrades -= src
+//using this line to seperate my sounds. I think of it seperate so i try to indent it when i cant but i cannot here
+		if (prob(80))
+			owner.playsound_local(owner.loc, 'sound/voice/blob/blobup1.ogg', 50, 1)
+		else if (prob(50))
+			owner.playsound_local(owner.loc, 'sound/voice/blob/blobup2.ogg', 50, 1)
+		else
+			owner.playsound_local(owner.loc, 'sound/voice/blob/blobup3.ogg', 50, 1)
```

Comment:
like we discussed, this is not how probability really works
also that comment line is weird and unnecessary

Reply:
This is intentional

100 -80 = 20
50% of /0 is 10.
So else if and else both have a 20%

Reply:
ok, comment still weird tho